ios/Wrappers/RNWrappers.m

```diff
@@ -187,26 +187,65 @@ + (JPUIConfiguration *)uiConfigurationFromConfiguration:(NSDictionary *)configur
     NSNumber *shouldDisplayAmount = [dictionary boolForKey:@"shouldDisplayAmount"];
-    
+
     uiConfiguration.isAVSEnabled = isAVSEnabled.boolValue;
     uiConfiguration.shouldDisplayAmount = shouldDisplayAmount.boolValue;
-    
+
     uiConfiguration.theme = [self themeFromUIConfiguration:dictionary];
-    
+
     return uiConfiguration;
 }
 
 + (JPTheme *)themeFromUIConfiguration:(NSDictionary *)uiConfiguration {
-    //TODO: Add theming configuration
-    return [JPTheme new];
+    JPTheme *theme = [JPTheme new];
+
+    NSDictionary *dictionary = [uiConfiguration optionalDictionaryForKey:@"theme"];
+
+    if (!dictionary) {
+        return theme;
+    }
+
+    theme.largeTitle = [UIFont fontWithName:[dictionary stringForKey:@"largeTitleFont"]
+                                       size:[[dictionary numberForKey:@"largeTitleSize"] doubleValue]];
+
+    theme.title = [UIFont fontWithName:[dictionary stringForKey:@"titleFont"]
+                                  size:[[dictionary numberForKey:@"titleSize"] doubleValue]];
+
+    theme.headline = [UIFont fontWithName:[dictionary stringForKey:@"headlineFont"]
+                                     size:[[dictionary numberForKey:@"headlineSize"] doubleValue]];
+
+    theme.headlineLight = [UIFont fontWithName:[dictionary stringForKey:@"headlineLightFont"]
+                                          size:[[dictionary numberForKey:@"headlineLightSize"] doubleValue]];
+
+    theme.body = [UIFont fontWithName:[dictionary stringForKey:@"bodyFont"]
+                                 size:[[dictionary numberForKey:@"bodySize"] doubleValue]];
+
+    theme.bodyBold = [UIFont fontWithName:[dictionary stringForKey:@"bodyBoldFont"]
+                                     size:[[dictionary numberForKey:@"bodyBoldSize"] doubleValue]];
+
+    theme.caption = [UIFont fontWithName:[dictionary stringForKey:@"captionFont"]
+                                    size:[[dictionary optionalNumberForKey:@"captionSize"] doubleValue]];
+
+    theme.captionBold = [UIFont fontWithName:[dictionary stringForKey:@"captionBoldFont"]
+                                        size:[[dictionary numberForKey:@"captionBoldSize"] doubleValue]];
+
+    theme.jpBlackColor = [UIColor colorFromHexString:[dictionary stringForKey:@"jpBlackColor"]];
+    theme.jpDarkGrayColor = [UIColor colorFromHexString:[dictionary stringForKey:@"jpDarkGrayColor"]];
+    theme.jpGrayColor = [UIColor colorFromHexString:[dictionary stringForKey:@"jpGrayColor"]];
+    theme.jpLightGrayColor = [UIColor colorFromHexString:[dictionary stringForKey:@"jpLightGrayColor"]];
+    theme.jpRedColor = [UIColor colorFromHexString:[dictionary stringForKey:@"jpRedColor"]];
+    theme.jpWhiteColor = [UIColor colorFromHexString:[dictionary stringForKey:@"jpWhiteColor"]];
+    theme.buttonColor = [UIColor colorFromHexString:[dictionary stringForKey:@"buttonColor"]];
+    theme.buttonTitleColor = [UIColor colorFromHexString:[dictionary stringForKey:@"buttonTitleColor"]];
```

Comment:
What about `button corner radius` and `back button custom image` ?

Reply:
Will add them now